swagger.yaml

```diff
@@ -4619,12 +4619,9 @@ definitions:
       name:
         type: string
         description: Product name of the device (for example "UHD Graphics 620 (Whiskey Lake)")
-      bus_info:
+      address:
```

Comment:
I haven't followed the work on the GPU support much so pardon me if this comment is not valid in this case.

Based on my understanding of how we are using `swagger.yaml` this change should be considered an API change. Is this correct? Do we have backward compatibility guarantees that we should keep?

This is all assuming the GPU support has been released already and it's supported. Otherwise, don't mind my comment :D

Reply:
There is no need for backward compatibility, as due to a bug in `assisted-installer-agent` GPU data has never been populated and stored anywhere.